ethers-contract/src/multicall/mod.rs

```diff
@@ -608,7 +608,7 @@ impl<M: Middleware> Multicall<M> {
     /// # }
     /// ```
     pub async fn call<D: Detokenize>(&self) -> Result<D, M> {
-        assert!(self.calls.len() < 16, "Cannot decode more than 16 calls");
+        assert!(self.calls.len() <= 16, "Cannot decode more than 16 calls");
```

Comment:
OK this should be fine due to this line https://github.com/gakonst/ethers-rs/blob/master/ethers-core/src/abi/tokens.rs#L130

Reply:
I have propotypes of both, will prepare another PR

Reply:
https://github.com/gakonst/ethers-rs/pull/1941